generator/metricbeat/Makefile

```diff
@@ -1,16 +1,14 @@
-BEAT_TYPE=metricbeat
-PREPARE_COMMAND=MODULE=elastic METRICSET=test make create-metricset ; FIELDS_FILE_PATH=module
-override FIELDS_FILE_PATH=
-export FIELDS_FILE_PATH
-
+BEAT_TYPE       = metricbeat
```

Comment:
I like the new indentation but thing I wonder is if the ` ` after the `=` could cause some issues? I remember we had a postfix space when it was something like  `BEAT_TYPE=metricbeat #is a comment`, not sure if it also applies to prefix.

Reply:
I'm pretty sure the spaces are dropped by make (in bash they would not be). I've used this syntax before without issues (https://github.com/elastic/golang-crossbuild/blob/master/Makefile.common and several other files in this repo).